activiti-spring-boot/spring-boot-samples/spring-boot-sample-migrate-users-groups-to-keycloack/src/main/java/org/activiti/tasks/MigrateUsersAndGroupsService.java

```diff
@@ -0,0 +1,15 @@
+package org.activiti.tasks;
+
+import org.activiti.engine.delegate.DelegateExecution;
+import org.activiti.engine.delegate.JavaDelegate;
+import org.springframework.stereotype.Component;
+
+@Component

```

Comment:
Presumably this class can be removed? Is it just a hangup from the previous implementation or is it necessary to use the engine in the test?